docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/link/link.twig

```diff
@@ -36,18 +35,32 @@
   {% endset %}
 {% endif %}
 
+{% if is_external %}
+  {% set external_icon %}
+    {% include '@atoms/icon/icon.twig' with {
+       symbol: 'upper-right-arrow',
+     } only %}
+  {% endset %}
+{% endif %}
+
 {% set link_text %}
   {% if icon and button_icon %}
     {% if icon_placement == 'before' %}
-      {{ button_icon }}{{ text|raw }}
+      {{ button_icon }}{{ text|raw }} {{ external_icon }}
     {% elseif icon_placement == 'after' %}
-      {{ text|raw }}{{ button_icon }}
+      {{ text|raw }}{{ external_icon }}{{ button_icon }}
     {% endif %}
   {% else %}
-    {{ text|raw }}
+    {{ text|raw }}{{ external_icon }}
   {% endif %}
 {% endset %}
 
+{% if is_new_window %}
+  {% set link_text %}
+    {{ link_text }} <span class="civictheme-link--is-new-window">Opens in a new tab/window.</span>
```

Comment:
`Opens in a new tab/window` - no dot

<img width="223" alt="_CIVIC-785__Component_-_Link_-_Accessibility_-_JIRA" src="https://user-images.githubusercontent.com/378794/188519215-f7141753-cfd8-43fa-99e3-65ac60038b5f.png">


in ACs - dot is outside of the textual value